includes/admin/class-amp-editor-blocks.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * AMP Editor Blocks extending.
+ *
+ * @package AMP
+ * @since 1.0
+ */
+
+/**
+ * Class AMP_Editor_Blocks
+ */
+class AMP_Editor_Blocks {
+
+	/**
+	 * Init.
+	 */
+	public static function init() {
+		if ( function_exists( 'gutenberg_init' ) ) {
+			$self = new self();
```

Comment:
I don't think this is right. Below you do `$editor_blocks = new AMP_Editor_Blocks()` so should not it just be using `$this` here instead of creating a second instance of itself?